UKHomeOffice/sgmr-service
# SPCR-219 Update remaining Cypress tests

## Description
- swapped localStorage to sessionStorage as that's what we use now
- updated tables to include titles in the expect as they exist in the text but are hidden on desktop (shown on mobile)
- update error class name
- add remove test data function
- comment out failing test on password reset as requires investigation into UUID issue (SPCR-222)

## To test
- pull down
- update `mailSlurpApiKey`
- run `npm run cypress:runner -- --env configFile=dev`
- run `edit-people.spec` test
- run `edit-vessel.spec` test
- run `my-account.spec` test
- run `password-reset.spec` test
- run `report-form-validation.spec` test
- run `sign-in.spec` test
- run `user-register.spec` test
- > these tests should pass


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
